test/integration/data-types.test.js

```diff
@@ -285,14 +285,15 @@ describe(Support.getTestDialectTeaser('DataTypes'), function() {
 
     return new Sequelize.Promise((resolve, reject) => {
       if (/^postgres/.test(dialect)) {
-        current.query(`SELECT extversion FROM pg_catalog.pg_extension WHERE extname='postgis';`)
-            .then((result) => {
-              if (result[0][0] && semver.lte(result[0][0].extversion, '2.1.7')) {
-                resolve(true);
-              } else {
-                resolve();
-              }
-            }).catch(reject);
+        current.query(`SELECT PostGIS_full_version();`)
+          .then((result) => {
+            const version = result[0][0].postgis_full_version.match(/^POSTGIS="(\d*.\d*.\d*)/)[1];
```

Comment:
The reason I changed it was that this command gives a different (correct) version number than the current one in master. For some reason they differ and I am not sure why. I just updated to use `PostGIS_lib_version` because its cleaner and maybe it will help?!